### PR TITLE
Monster Spells and Necrotic nerfs

### PR DESCRIPTION
**Pull request by RobbieNeko**
#### Summary

SUMMARY: Mods "Made Krabgeks and Black puddings use monster versions of spells, rather than the player versions. Nerfed Necrotic Gaze."

#### Purpose of change

As noted by Volch on the Discord, Krabgeks used the Necrotic Gaze spell, and the sheer power of the spell will easily kill any player unfortunate enough to encounter one in close range. As such, moved the Krabgek to a much less powerful (though still buffed compared to the old strength) monster-only version of Necrotic Gaze. The same has pre-emptively been applied to the Black Pudding and its acidic spray, even though the acidic spray was much less of an issue.

Due to #3193 , I also nerfed Necrotic gaze by cutting its min damage in half. As it was, it was much too good.

#### Describe the solution

- Migrated Krabgek and Black Pudding to using monster-only versions of their relevant spells
- Cut Necrotic Gaze's minimum damage in half, subtracted the value removed from its maximum damage.

#### Describe alternatives you've considered

- Keeping the monsters using the player versions of the spells, and balancing it around the monsters being able to use it.
Rather defeats the point of my previous spell buffs, and would result in those spells simply being less powerful than their non-monster cousins.

- Deciding that the overly strong Krabgeks in melee are a feature, not a bug
Krabgeks were not designed with such intentions in mind. I will very much so consider adding in some monsters to specifically counter melee and ranged individually in the future, though.

- Leaving Necrotic Gaze be
Valid, but at the moment it deserves a bit of a nerf to make it reasonable. When the issue with there being no resistance / immunity to Bio damage is fixed, then I'll gladly consider buffing it back to a similar state as now.

#### Testing

Made sure the game didn't yell at me or explode, and confirmed that Necrotic Gaze had its damage correctly lowered.

#### Additional context

This is mostly a hotfix, the next proper PR for Magiclysm will be the content addition I intended.

### CODE REVIEW

**Comment by RobbieNeko** [comment]
... why in the world is it throwing a fit about my spacing in krab_gaze?

File: data/mods/Magiclysm/Spells/monsterspells.json [hunk 1]
```diff
@@ -129,11 +129,53 @@
...
     "base_energy_cost": 150,
     "energy_source": "MANA",
-    "difficulty": 2,
     "damage_type": "fire"
+  },
+  {
+    "id": "krab_gaze",
+    "type": "SPELL",
+    "name": { "str": "Krabgek Gaze" },
+    "description": "Carcinization ray, go! (Krabgek version of Necrotic Gaze for balance purposes)",
```
Comment (chaosvolt) [inline data/mods/Magiclysm/Spells/monsterspells.json:141]:
```suggestion
    "description": "Carcinization ray, go!  (Krabgek version of Necrotic Gaze for balance purposes)",
```
Probably sees that as equivalent to two separate sentences, and description text tends to mandate double spaces between sentences for style reasons.

File: data/mods/Magiclysm/Spells/monsterspells.json [hunk 2]
```diff
@@ -129,11 +129,53 @@
...
+    "max_level": 37,
+    "base_casting_time": 100,
+    "base_energy_cost": 2,
+    "damage_type": "bio"
+  },
+  {
+    "id": "ooze_acidicspray",
+    "type": "SPELL",
+    "name": { "str": "Ooze Acidic Spray" },
+    "description": "Somone's pudding recipe has gone *very* awry... (Monster Spell)",
```
Comment (chaosvolt) [inline data/mods/Magiclysm/Spells/monsterspells.json:161]:
```suggestion
    "description": "Somone's pudding recipe has gone *very* awry...  (Monster Spell)",
```
Same here

Comment (RobbieNeko) [reply data/mods/Magiclysm/Spells/monsterspells.json:161]: It actually wasn't angry about that spacing, it was saying that an ellipses character is preferred to 3 dots.

File: data/mods/Magiclysm/Spells/monsterspells.json (hunk 1 above)
Comment (RobbieNeko) [reply data/mods/Magiclysm/Spells/monsterspells.json:141]: ... that is very strange. I've never done double-spacing after a sentence before.

File: data/mods/Magiclysm/Spells/monsterspells.json (hunk 2 above)
Comment (chaosvolt) [reply data/mods/Magiclysm/Spells/monsterspells.json:161]: Oh right I derped and didn't see that, thanks. XD

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**